flutter_frontend_server/test/to_string_test.dart

```diff
@@ -7,10 +7,9 @@ import 'dart:io';
 import 'package:frontend_server/frontend_server.dart' as frontend
     show ProgramTransformer, ToStringTransformer, ToStringVisitor;
 import 'package:kernel/kernel.dart';
-import 'package:mockito/mockito.dart';
 import 'package:path/path.dart' as path;
 
-import 'package:test/test.dart';
+import 'utils.dart';
```

Comment:
FWIW, both `mockito` and `test` are available in `//third_party/dart/third_party/pkg`.

But getting rid of mockito will vastly simplify NNBD migration.

Reply:
Yeah, nnbd is why I wanted to get rid of mockito. Getting rid of package:test has two benefits.
1. Many fewer path dependencies to explicitly override in the pubspec.yaml.
2. And therefore less chance that the dependencies will have version conflicts so that we can use `pub get --offline` to generate the `package_config.json`.